src/main/java/com/notably/model/suggestion/SuggestionModel.java

```diff
@@ -29,13 +29,13 @@
      * TODO: add logo.
      * @return The meaning of the user input.
      */
-    Property<Optional<String>> commandTextProperty();
+    Property<Optional<String>> getCommandTextProperty();
```

Comment:
This should be called `commandTextProperty` by convention!

Reply:
![image](https://user-images.githubusercontent.com/38042032/76855255-b71b8b80-688b-11ea-942f-14b3414fca2d.png)

Reply:
https://docs.oracle.com/javafx/2/binding/jfxpub-binding.htm